src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/AesCngCryptoTransform.cs

```diff
@@ -72,6 +73,29 @@ protected override void Reset()
             }
         }
 
+        private static SafeAlgorithmHandle GetCipherAlgorithm(CipherMode cipherMode)
```

Comment:
Consider naming this GetCachedCipherAlgorithm, or something that hints that this can only return algorithms that are rooted and kept alive, since the caller's not doing that.